draft-ietf-tls-tls13.md

```diff
@@ -5924,6 +5924,21 @@ server-to-client encryption keys because that would entail
 the reuse of those keys. This parallels the use of the early
 application traffic keys only in the client-to-server direction.
 
+## Static (EC)DH Shares
+
+For various reasons, some implementations may be tempted to use static
+(EC)DH private keys. Using such keys lowers the security guarantees of
+TLS 1.3. Adversaries that get access to the static (EC)DH private key
```

Comment:
Lowering a guarantee feels like an odd phrasing to me, as a guarantee is supposed to be a fixed thing.  So, one might reduce the level of security that is guaranteed, or break a guarantee, or similar, instead.